src/components/TodoList.js

```diff
@@ -0,0 +1,25 @@
+/* @jsx createElement */
+import { createElement } from '../lib/React';
+import { useSelector } from '../lib/Redux';
+import TodoListItem from './TodoListItem';
+
+const TodoList = () => {
+  const {
+    todo: { type, todos },
+  } = useSelector();
+  return (
+    <ul id="todo-list" className="todo-list">
+      {todos
+        .filter((todo) => {
+          if (type === 'all') return true;
+          if (type === 'todo') return !todo.isCompleted;
+          if (type === 'completed') return todo.isCompleted;
+        })
+        .map((todo) => (
```

Comment:
type === 'all'일 때는 filter를 사용하지 않아도 되도록 이런 느낌의 구조는 어떨지 제안드려봅니다!
```suggestion
      let filteredTodos;

      if (type === 'all') {
        filteredTodos = todos;
      } else if (type ==='todo') {
        // filteredTodos = todos.filter((todo) => ...)
      }
    
      return (
        <ul id="todo-list" className="todo-list">
          {filteredTodos
            .map((todo) => (
```

Reply:
이런 방법도 있군요! 참고하겠습니다!!😀😀